node/bin/index.js

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env node
+
+const run = require('../src/env0-deploy-cli');
+
+run();
```

Comment:
missing newline at the end of the file

Reply:
Actually when I moved `index.js` into `bin/` then it stopped working for me 🤷 
so I moved it back into root: 290005d and added that missing newline 😉